mountinfo/lookup_linux_test.go

```diff
@@ -0,0 +1,40 @@
+// +build linux
+
+package mountinfo
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/containerd/containerd/testutil"
+	"github.com/stretchr/testify/assert"
+)
+
+func testLookup(t *testing.T, fsType string) {
+	testutil.RequiresRoot(t)
+	mnt, err := ioutil.TempDir("", "containerd-mountinfo-test-lookup")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(mnt)
+	dev := testutil.NewTestDevice(t, testutil.TestDeviceOpts{
+		Mountpoint: mnt,
+		MKFS:       []string{"mkfs", "-t", fsType},
+		Size:       100 << 20, // 100MB
+	})
+	defer dev.Remove(t)
+	info, err := Lookup(mnt)
+	if err != nil {
+		t.Fatal(err)
+	}
+	assert.Equal(t, fsType, info.FSType)
```

Comment:
Could you also test device has prefix `/dev/loop` (I think we can always rely on that being the case for Linux) and the mountpoint